src/renderer/components/execute/Results.tsx

```diff
@@ -30,36 +144,49 @@ const Results = () => {
         <div>
           <h1>Execution Results</h1>
         </div>
-        <Table className="table">
-          <thead>
-            <tr key={results.id}>
-              <th>Name</th>
-              <th>ID</th>
-              <th>Database</th>
-              <th>Type</th>
-              <th>Result</th>
-              <th> Message </th>
-            </tr>
-          </thead>
-          <tbody>
-            {results.rules.map((value) => {
-              return (
-                <>
-                  {passFail(value) ? (
-                    <AllPass Rule={value} />
+        {execution.rules.map((rule) => {
+          return (
+            <Accordion className="accordionClass">
+              <AccordionItem eventKey="0">
+                <AccordionHeader id="header">
+                  {passFail(rule) ? (
+                    <>
+                      {rule.name} <CheckLg className="Check" />
+                    </>
                   ) : (
-                    <Displaytests Rule={value} />
+                    rule.name
                   )}
-                </>
-              );
-            })}
-          </tbody>
-        </Table>
+                </AccordionHeader>
+                <AccordionBody>
+                  {rule.unitTests.map((Test) => {
+                    return (
+                      <Table className="Results-Table">
+                        <tr key={Test.id}>
+                          <td>{Test.name}</td>
+                          {Test.result ? (
+                            <td> test Passed </td>
+                          ) : (
+                            <td> Test Failed</td>
+                          )}
+                          {Test.result ? <CheckLg /> : <XLg />}
```

Comment:
Maybe we should center the checks / Xs vertically and align them horizontally with each other?

<img width="77" alt="Screen Shot 2022-11-04 at 1 40 08 PM" src="https://user-images.githubusercontent.com/29695042/200040685-b8acb111-36de-4bb0-bd8c-aa4689141bb6.png">